core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -420,6 +420,29 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val STORAGE_DECOMMISSION_SHUFFLE_BLOCKS_ENABLED =
+    ConfigBuilder("spark.storage.decommission.shuffleBlocks.enabled")
```

Comment:
need to update configuration.md for these configs.   I think the one above is missing too, so could do in followup, unless you are specifically trying to leave undocumented?

Reply:
I was planning on saving that for once we've agreed it's ready for general usage. I know the SPIP is approved, but I still view this as more of a developer feature (e.g. one we would expect a cloud vendor to build on top of) than ready for end user feature. What do you think?

Reply:
ok, that is fine with me. just wanted to make sure we thought about it.